lib/active_interactor/organizer/interactor_interface.rb

```diff
@@ -69,6 +70,10 @@ def perform(target, context, fail_on_error = false, perform_options = {})
         return if check_conditionals(target, :if) == false
         return if check_conditionals(target, :unless) == true
 
+        options = self.perform_options.merge(perform_options)
```

Comment:
this is violating the useless assignment cop because its duplicated on [line 78](https://github.com/aaronmallen/activeinteractor/pull/350/files#diff-00757cc6878a3714635b5d2efabd82c8ea7e6a006ec0e9b73a1d1283944bba56R78)

Reply:
Ah good catch